java/src/main/java/com/logicalclocks/hsfs/metadata/FeatureGroupBase.java

```diff
@@ -116,6 +115,11 @@ public FeatureGroupBase(FeatureStore featureStore, Integer id) {
     this.id = id;
   }
 
+  public void setFeatures(List<Feature> features) {
```

Comment:
Can we have at least a unit test so when someone goes in and doesn't know why we are doing this, doesn't remove it?